### PR TITLE
Fixes follow reset bug #220

### PR DESCRIPTION
**Pull request by brendanheywood**

### CODE REVIEW

File: src/L.Control.Locate.js [hunk 1]
```diff
@@ -482,10 +482,12 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
                     f.bind(this._map)([this._event.latitude, this._event.longitude]);
                 } else {
                     var f = this.options.flyTo ? this._map.flyToBounds : this._map.fitBounds;
+                    this._ignoreEvent = true;
```
Comment (domoritz) [inline src/L.Control.Locate.js:499]: Please add a comment about why this is necessary.

**Review by domoritz — CHANGES_REQUESTED** [review]

**Comment by brendanheywood** [comment]
Hmm this fix isn't good enough, depending on the timing of the requestAnimationFrame the subsequent zoom and zoom event could be async in the next tick which explains why this was intermittent.

**Comment by brendanheywood** [comment]
Ok this should be pretty solid now. In my research I also found another concept:

https://stackoverflow.com/questions/31992278/detect-user-initiated-pan-zoom-operations-on-leaflet

But that e.hard attribute was never documented and has since been removed from leaflet core

**Comment by domoritz** [comment]
Thank you!

**Pull request merged**